### DISCUSSION TITLE
Anyone used kitten ssh for gcloud private VM yet?

### DISCUSSION THREAD

**Discussion by sytranvn** [kovidgoyal/kitty#7133] [Q&A]
I have a few VMs sit behind cloud private network and often need to use `gcloud compute ssh` command to access the machines. So I have to open new session if I want to split window. And it does not work well with kitty because lacking of terminfo. But installing terminfo again and again for each VM is not ideal (I need to access many of them). If you figured out a way to use kitty nicely with gcloud, please share your tips. Thank you.

**Comment by sytranvn** [comment]
`TERM=xterm-256color gcloud compute ssh` to works for me.

**Reply by ericrbg-harmonic** [reply]
is there a more principled solution? it would be nice for the proper terminfo files to be copied over
